src/components/EnrolmentOfficerFormPanel.js

```diff
@@ -33,14 +44,14 @@ const EnrolmentOfficerFormPanel = (props) => {
     `,
       { system_id: 1 }, // EO System Role is 1
     );
-    const isValid = !isLoading
+    const isValid = !isLoading;
     useEffect(() => {
-      if (isValid & isEnabled & !has_role) {
-        const role = data.role.edges[0].node
-        const roles  = !!edited.roles ? edited.roles : [];
-        roles.push(role)
-        edited.roles = roles
-        onEditedChanged(edited)
+      if (isValid & isEnabled & !hasRole) {
+        const role = data.role.edges[0].node;
+        const roles = !!edited.roles ? edited.roles : [];
```

Comment:
Can we try with 'optional chaining'? 
`!!edited?.roles ?? [];`
I'm not sure if this could work in this context but let's try.

Reply:
Because `EnrolmentOfficerFormPanel.js` and `ClaimAdministratorFormPanel.js` is changed in scope of the [OP-952](https://openimis.atlassian.net/browse/OP-952) and [OP-1055](https://openimis.atlassian.net/browse/OP-1055) I will fix it in [OP-952/OP-1055 PR](https://github.com/openimis/openimis-fe-admin_js/pull/35).